src/sage/rings/function_field/order_polymod.py

```diff
@@ -1325,8 +1325,8 @@ def bar(I): # transfer to O/pO
 
             def liftb(Ib):
                 m = [vector([fr(e) for e in v]) for v in Ib]
-                m += [v for v in pO._hnf]
-                return self._ideal_from_vectors_and_denominator(m,1)
+                m.append(list(pO._hnf))
```

Comment:
Are you sure it's the right form ? I think it should be `m.extend(pO._hnf)`.

Reply:
oops, oui, désolé